pages.es/linux/man.md

```diff
@@ -3,30 +3,34 @@
 > Da formato y muestra páginas del manual.
 > Más información: <https://manned.org/man>.
 
-- Muestra la página del manual para un comando:
+- Muestra la página del manual de un comando:
 
 `man {{comando}}`
 
-- Muestra la página del manual para un comando de la sección 7:
+- Abre la página del manual de un comando en un navegador (requiere que la variable `BROWSER` esté establecida):
+
+`man --html {{command}}`
+
+- Muestra la página del manual de la sección 7 de un comando:
 
 `man {{7}} {{comando}}`
 
 - Lista todas las secciones disponibles para un comando:
 
 `man --whatis {{comando}}`
 
-- Muestra las rutas usadas para la búsqueda de las páginas:
+- Muestra las rutas usadas en la búsqueda de las páginas:
 
 `man --path`
 
 - Muestra la ubicación de la página del manual en lugar de la propia página:
 
 `man --where {{comando}}`
 
-- Muestra la página del manual usando una ubicación específica:
+- Muestra la página del manual usando un idioma (locale) específico (p.e. es para español):
 
-`man --locale={{locale}} {{comando}}`
+`man --locale={{idioma}} {{comando}}`
```

Comment:
```suggestion
`man --locale {{idioma}} {{comando}}`
```

Reply:
I just reviewed, sorry about that, the = somehow went through.  Just fixed it. Thank you again.